src/utils/functions.ts

```
@@ -70,3 +71,52 @@ export const isObjectEmpty = (object: object) => object && Object.keys(object).l
+    if (Number.isNaN(current)) {
+        return false;
+    }
+    let order = null;
+
+    for (let i = 1; i < array.length; i++) {
+        const nextOne = toNumber(array[i]);
+        if (Number.isNaN(nextOne)) {
+            return false;
+        }
+        if (current === nextOne) {
+            continue;
```

ESLint

[error] Unexpected use of continue statement.

(no-continue)